subprojects/core/src/integTest/groovy/org/gradle/api/ExternalScriptExecutionIntegrationSpec.groovy

```diff
@@ -17,15 +17,20 @@ package org.gradle.api
 
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.test.fixtures.server.http.HttpServer
+import spock.lang.Unroll
 
 class ExternalScriptExecutionIntegrationSpec extends AbstractIntegrationSpec {
```

Comment:
Yes, I wasn't thinking about credentials in the URL.  We should probably try that too.

I was thinking of _proxy_ credentials with HTTPS and if that worked or what it looked like when that went wrong.